pipe_segment/transform/segment.py

```diff
@@ -42,17 +43,25 @@ def __init__(self, segments_pcoll_kv,
 
     @staticmethod
     def _convert_messages_in(msg):
-        msg = dict(msg)
-        msg['timestamp'] = datetimeFromTimestamp(msg['timestamp'])
-        msg['mmsi'] = msg['ssvid']
-        type = msg.get('type')
-        if type is not None:
-            msg['old_type'] = type
-            if (type.startswith('AIS.')):
+        def convert_type (t):
```

Comment:
@pwoods25443 will the segmenter work when there is no type? the use case are VMS pipes, until know they didn't have type on the normalization step.

Reply:
@enriquetuya the idea here is that if there is a type field present, and it is an AIS type that is identifiable as type A or B, then we use the type to inform which segment an identity message (type 5 or 24) gets added to.  If we have a lat, lon present then this is not used, and if the type is anything other than AIS 1,2,5,18,19,24 then it is not used. In all cases where the type is not used, the segmented behaves as it did before